src/pages/javascript/switch-statements/index.md

```diff
@@ -2,14 +2,71 @@
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/switch-statements/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Syntax:
+```javascript
+switch(expression) {
+    case 1:
+        console.log('1');
+        break;
+    case 2:
+        console.log('2');
+        break;
+    default:
+        console.log('No true condition, default');
+}
+```
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+The above snippet shows the syntax for a basic `switch` statement. In this example, there are 3 different scenarios for:
+- `expression = 1`: First condition is true, and `1` gets printed to the console.
+- `expression = 2`: Second condition is true, and `2` gets printed to the console.
+- `expression = 'anything else'`: Case 1 and Case 2 are both false, so the default condition is executed.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+The `default` condition is a condition that will be executed if none of the other cases are true.
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+### Break
+The `break` keyword is required in each case to make sure that only the code in that case gets executed. Without the break, multiple cases could be executed. When JavaScript reaches a `break` keyword, it breaks out of the switch block. If the `break` was left out of the above example, this is what would happen:
+
+```javascript
+var expression = 1;
+switch(expression) {
+    case 1:
+        console.log('1');  // 1 would be printed to console
+    case 2: // As there is no break after case 1, this case is also executed.
+        console.log('2'); // 2 would be printed to the console.
+        break; // break -> Switch statement is exited
+    default:
+        console.log('No true condition, default');
+}
+```
 
 
+### Execute multiple cases:
+`switch` statements also allow for the same code block to be executed by multiple cases. This can be done by adding 1 or more `case :` keywords before a code block.
+
+Eg:
+```javascript
+switch (day) {
+    case 4:
+    case 5:
+        console.log('it is nearly the weekend');
+        break; 
+    case 0:
+    case 6:
+        console.log('it is the weekend');
+        break;
+    default: 
+       console.log('Looking forward to the Weekend');
+}
+```
+
+In the above snippet:
+- If `day is 4 or 5` (Thursday or Friday), `it is nearly the weekend` will be printed to the console as the first case gets executed.
```

Comment:
Minor comment for this and the next bullet - let's remove the "is" and "or" from the code highlighting, so it reads: "If `day` is `4` or `5` (Thursday or Friday)..."

Reply:
Fixed, thanks!